Content.Shared/GameTicking/SharedGameTicker.cs

```diff
@@ -108,10 +108,10 @@ public sealed class TickerJobsAvailableEvent : EntityEventArgs
         /// <summary>
         /// The Status of the Player in the lobby (ready, observer, ...)
         /// </summary>
-        public Dictionary<StationId, Dictionary<string, int>> JobsAvailableByStation { get; }
-        public Dictionary<StationId, string> StationNames { get; }
+        public Dictionary<EntityUid, Dictionary<string, uint?>> JobsAvailableByStation { get; }
+        public Dictionary<EntityUid, string> StationNames { get; }
```

Comment:
again dicts with uids as the key are sus

Reply:
StationId no longer exists and the station entities do not exist on the client, what do?